scene/gui/tree.cpp

```diff
@@ -247,31 +247,25 @@ void TreeItem::_propagate_check_through_parents(int p_column, bool p_emit_signal
 		return;
 	}
 
-	bool all_unchecked_and_not_indeterminate = true;
-	bool any_unchecked_or_indeterminate = false;
+	bool has_checked = false;
+	bool has_unchecked = false;
+	bool has_indeterminate = false;
 
 	TreeItem *child_item = current->get_first_child();
-	while (child_item) {
-		if (!child_item->is_checked(p_column)) {
-			any_unchecked_or_indeterminate = true;
-			if (child_item->is_indeterminate(p_column)) {
-				all_unchecked_and_not_indeterminate = false;
-				break;
-			}
+	while(child_item) {
```

Comment:
```suggestion
	while (child_item) {
```
Code style, the code you replaced was correct already

Reply:
Sorry for this I didn't notice.

Reply:
> I didn't notice

@Ymanawat You'd `clang-format` to avoid that. See [Code style guidelines](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html) for more info (I recommend pre-commit hooks for automatic formatting checking etc., they're mentioned in there).